opal/mca/btl/ofi/btl_ofi_providers.h

```diff
@@ -0,0 +1,42 @@
+/* -*- Mode: C; c-basic-offset:4 ; indent-tabs-mode:nil -*- */
+/*
+ * Copyright (c) 2018      Intel, Inc, All rights reserved
+ * Copyright (c) 2018      Los Alamos National Security, LLC. All rights
+ *                         reserved.
+ *
+ * $COPYRIGHT$
+ *
+ * Additional copyrights may follow
+ *
+ * $HEADER$
+ */
+
+#include "btl_ofi.h"
+
+#define MCA_BTL_OFI_REQUIRED_CAPS       (FI_RMA | FI_ATOMIC)
+#define MCA_BTL_OFI_REQUESTED_MR_MODE   (FI_MR_UNSPEC)
+
+
+/* validate information returned from fi_getinfo().
+ * return OPAL_ERROR if we dont have what we need. */
+static int validate_info(struct fi_info *info)
```

Comment:
Why is this in a header file?  It doesn't seem like this needs to be inlined.

Reply:
maybe we dont need this file anymore after I removed the provider hard-coded setting.  Let me move this onto component.c.